src/pip/_internal/vcs/subversion.py

```diff
@@ -37,7 +54,57 @@ def should_add_vcs_url_prefix(cls, remote_url):
     def get_base_rev_args(rev):
         return ['-r', rev]
 
+    def vcs_global_options(self):
+        # type: () -> List[str]
+        """Options to be used on calls to Subversion that contact the server.
+
+        These options are applicable for the following ``svn`` subcommands used
+        in this class.
+
+            - checkout
+            - export
+            - info
+            - switch
+            - update
+
+        :return: A list of command line arguments to pass to ``svn``.
+        """
+        options = []  # type: List[str]
+        if not self.use_interactive:
+            # --non-interactive switch is available since Subversion 0.14.4.
+            options.append('--non-interactive')
```

Comment:
If you return here, you don't need an else clause below (you can unindent).

Reply:
I probably need to untrain myself from avoiding early returns. There was some `clang-tidy` lint I used to adhere that would complain about the readability of early return, but I agree with you that the fixed up version is easier to follow.